apps/src/code-studio/progress.js

```diff
@@ -348,6 +349,9 @@ function initializeStoreWithSections(store, scriptData) {
       };
```

Comment:
(This comment is only partially related to this change.)  I was slightly confused reading this the first time because the comment on 340-341 now only applies to the code from 344-350.  My suggestion is to move the google classroom-specific call to a separate if block below the current `if (currentSection)` block.  This also avoids the duplicated `store.dispatch` call.

Reply:
@jamescodeorg let me know if you think this is clearer now. i wasn't able to avoid the duplicated dispatch call since the call to `selectSection` has to happen after it, but when i went to move the google stuff into its own block, that highlighted that most of what we're doing here is on relevant if we have a current section. @jmkulwik and i were just discussing a similar case up at line 307 and decided the logic is clearer if we make that explicit, so i updated the flow here to match what she did there.